### PR TITLE
BugFixes-2 and Doc Inputs

### PR DESCRIPTION
**Pull request by cs-rohit-khune**
### Mantis#0931227
- Validated that the mitre records that are part of the "Created Records ID" field in the scenario record are not deleted during the scenario reset.

### Docs
- Updated Release notes
- Updated usage.md

### CODE REVIEW

File: playbooks/02 - Use Case - SOC Simulator/Reset Scenario.json [hunk 1]
```diff
@@ -49,13 +49,14 @@
                     "item": "{{vars.demoRecordIRIs}}",
                     "__bulk": false,
                     "parallel": true,
-                    "condition": ""
+                    "condition": "{{'mitre' not in vars.item}}"
```
Comment (cs-suyog-jinturkar) [inline playbooks/02 - Use Case - SOC Simulator/Reset Scenario.json:52]: make it a list in set variable and user here. so in future, we will need to add items in the list.

File: release_notes.md [hunk 2]
```diff
@@ -2,28 +2,21 @@
...
 - Following playbooks have been modified
 
   - **Run Selected Scenario** playbook modifications:
-
-    - The loop execution mode of the *Create Alert* step is updated from `Sequential` to `Parallel`
-
-    - A `Simulation` tag is added to the playbook
-
-    - The **Create Scenario** step is renamed to **Create Alerts** 
+    - A `SystemWaitForCompletion` tag is added to the playbook
```
Comment (cs-suyog-jinturkar) [inline release_notes.md]: why added?

File: release_notes.md [hunk 3]
```diff
@@ -2,28 +2,21 @@
...
 
   - **Run Scenario** playbook modifications:
-
-    - The loop execution mode of the *Create Alert* step is updated from `Sequential` to `Parallel`
+    - A `SystemWaitForCompletion` tag is added to the playbook
 
   - **Run Scenario - Create Alerts** playbook modifications:
+    - Updated the logic that displays the newly created demo record IRIs in scenario record's "Created Records ID" field.
```
Comment (cs-suyog-jinturkar) [inline release_notes.md]: Can we mentioned as "fixed inconsistencies in demo record IRIs in Scenario module"

File: release_notes.md [hunk 4]
```diff
@@ -2,28 +2,21 @@
...
-    - The loop execution mode of the *Create Alert* step is updated from `Sequential` to `Parallel`
+    - A `SystemWaitForCompletion` tag is added to the playbook
 
   - **Run Scenario - Create Alerts** playbook modifications:
+    - Updated the logic that displays the newly created demo record IRIs in scenario record's "Created Records ID" field. 
 
-    - Updated the playbook so it now adds post-create comments in scenario records after demo records creation
+  - **Reset Scenario** playbook modifications:
+    - A `SystemWaitForCompletion` tag is added to the playbook
+    - Updated the playbook so it excludes mitre records from deletion
```
Comment (cs-suyog-jinturkar) [inline release_notes.md:18]: Exclude recordtype list support has been added to exclude said record types while reseting the scenario.

File: release_notes.md (hunk 2 above)
Comment (cs-rohit-khune) [reply release_notes.md]: This is to make the "Simulate Scenario" button in the Scenario record, to be disabled for the long-running scenario playbook. In current behaviour, the "Simulate Scenario" button remains active, leading to confusion at users and user may end up executing same scenario twice.

File: release_notes.md (hunk 4 above)
Comment (cs-rohit-khune) [reply release_notes.md:18]: No, we cannot mention this as these changes are not generic and are hardcoded only for Mitre-related modules

File: release_notes.md (hunk 3 above)
Comment (cs-rohit-khune) [reply release_notes.md]: Made changes, please check

**Pull request merged**